assets/js/src/bulk-delete-post-type.js

```diff
@@ -0,0 +1,17 @@
+jQuery( document ).ready( function () {
+	jQuery( '.select2-post' ).select2( {
+		width: '300px',
+		templateSelection: formatLabel
+	} );
+});
+
+function formatLabel (state) {
```

Comment:
Make this as an inline function. Otherwise we might get function name collusion if other plugins have the same name.